ORGS.md

```diff
@@ -0,0 +1,26 @@
+# Samvera GitHub Organizations
+Samvera Community projects are currently organized between the following three GitHub Organizations:
+
+- https://github.com/samvera
+- https://github.com/samvera-labs
+- https://github.com/samvera-deprecated
+
+## Samvera
+
+Currently, projects under active maintenance (including the Core Components) are managed within this Organization. Aside from the Core Components, this consists of the following:
+
+| Project | Description |
+|-----|-----|
+| [Hyku](https://github.com/samvera/hyku) | About Hyku: A multi-tenant Hyrax application built on the latest and greatest Samvera community components. Brought to you by the Hydra-in-a-Box project partners and IMLS; maintained by the Hyku Interest Group. |
+| [Hyrax](https://github.com/samvera/hyrax) | Hyrax is a Ruby on Rails Engine built by the Samvera community. Hyrax provides a foundation for creating many different digital repository applications. |
+| [node-iiif](https://github.com/samvera/node-iiif) | This module provides a full-featured IIIF Image API 2.1 image processor. It covers only the image processing pipeline itself, leaving input and output to the caller. |
+| [serverless-iiif](https://github.com/samvera/serverless-iiif) | IIIF Image API 2.1 server in an AWS Serverless Application |
```

Comment:
I thought with the promotion to the `samvera` org these were now considered core components.  Is that not the case?